src/mn_setup.c

```diff
@@ -2213,9 +2237,6 @@ setup_menu_t comp_settings1[] = {
 
     {"Compatibility-breaking Features", S_SKIP | S_TITLE, M_X, M_SPC},
 
-    {"Direct Vertical Aiming", S_ONOFF | S_STRICT, M_X, M_SPC,
-     {"direct_vertical_aiming"}, .action = P_UpdateDirectVerticalAiming},
-
     {"Auto Strafe 50", S_ONOFF | S_STRICT, M_X, M_SPC, {"autostrafe50"},
```

Comment:
There's only one item here now (Auto Strafe 50). Should we remove the "compat breaking" title and extra `MI_GAP`? Or leave it alone?

Reply:
Thanks, LGTM. I'll close the other PR.